src/components/ListView/ListViewItem.js

```diff
@@ -39,18 +42,21 @@ class ListViewItem extends React.Component {
       leftContent,
       checkboxInput,
       hideCloseIcon,
+      compoundExpand,
+      compoundExpanded,
+      onCloseCompoundExpand,
       ...other
     } = this.props;
     const { expanded } = this.state;
 
     if (children) {
-      return (
-        <ListViewGroupItem expanded={expanded} stacked={stacked} {...other}>
-          <ListViewGroupItemHeader toggleExpanded={() => this.toggleExpanded()}>
-            <ListViewExpand
-              expanded={expanded}
-              toggleExpanded={() => this.toggleExpanded()}
-            />
+      if (compoundExpand) {
+        return (
+          <ListViewGroupItem
+            expanded={compoundExpanded}
+            stacked={stacked}
+            {...other}
```

Comment:
This looks like the issue related to the warning I was seeing... just need to filter these so they're specific to the child ListViewGroupItem.